src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -68,6 +62,10 @@ def set_image
         flash[:error] = "KIWI image '#{params[:id]}' does not exist"
         redirect_back(fallback_location: root_path)
       end
+
+      def set_repositories
+        @repositories = @image.repositories.order(:order)
```

Comment:
So looks like you could use a model spec to make sure the model returns them ordered and either a controller spec or a feature spec to make sure they are shown in the right order.